cloudflare/resource_cloudflare_load_balancer.go

```diff
@@ -136,6 +144,149 @@ func resourceCloudflareLoadBalancer() *schema.Resource {
 	}
 }
 
+var rulesElem = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"name": {
+			Type:         schema.TypeString,
+			Required:     true,
+			ValidateFunc: validation.StringLenBetween(1, 200),
+		},
+
+		"priority": {
+			Type:     schema.TypeInt,
+			Optional: true,
+			Computed: true,
+			Default:  nil,
```

Comment:
are these `Default`s all necessary? this one in particular might be problematic as the zero value for int is 0, notnil.